Packs/PiHole/Integrations/PiHole/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```

Comment:
please remove changelog file, its not needed

Reply:
Deleted as requested.